bigtable/integration_test.go

```diff
@@ -1148,6 +1148,73 @@ func TestIntegration_SampleRowKeys(t *testing.T) {
 	}
 }
 
+func TestIntegration_TableDeletionProtection(t *testing.T) {
+	testEnv, err := NewIntegrationEnv()
+	if err != nil {
+		t.Fatalf("IntegrationEnv: %v", err)
+	}
+	defer testEnv.Close()
+
+	timeout := 2 * time.Second
+	if testEnv.Config().UseProd {
+		timeout = 5 * time.Minute
+	}
+	ctx, _ := context.WithTimeout(context.Background(), timeout)
+
+	adminClient, err := testEnv.NewAdminClient()
+	if err != nil {
+		t.Fatalf("NewAdminClient: %v", err)
+	}
+	defer adminClient.Close()
+
+	tblConf := TableConf{
+		TableID: myTableName,
+		Families: map[string]GCPolicy{
+			"fam1": MaxVersionsPolicy(1),
+			"fam2": MaxVersionsPolicy(2),
+		},
+		DeletionProtection: Protected,
+	}
+
+	if err := adminClient.CreateTableFromConf(ctx, &tblConf); err != nil {
+		t.Errorf("create table from config: %v", err)
+	}
+
+	table, err := adminClient.getTable(ctx, myTableName, btapb.Table_FULL)
```

Comment:
Please check err. It will give us more details if some is wrong with getTable.

Why can't we use TableInfo instead here?